packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management.psd1

```diff
@@ -21,9 +21,9 @@
 
 
 @{
-ModuleVersion = '2.3.0'
+ModuleVersion = '3.0.0'
 
-GUID = 'dd9cad34-ad03-439b-b347-590625302c38'
+GUID = '2a3e34b4-5564-488e-aaf6-f2cba3f7f05d'
```

Comment:
What is it that means you need to update this GUID? If it's the version update, than maybe split this change and the one above into a separate commit for clarity and to help us figure this out in future.

Actually the version should be injected rather than hard-coded (I'll create a card for that). Should we also generate the GUID?